tests/integration_tests/test_message_queue_service.py

```diff
@@ -109,7 +107,7 @@ async def test_incorrect_vhost(mocker, caplog):
 
 
 async def test_parse_incoming_message(mq_service, consumer):
-    payload = {"msg": "test message", "another_key": "value"}
```

Comment:
This is just a temporary change to show that messages from earlier tests are still lingering in the list of received messages. I'm not sure why this started t o happen. It is probably related to some dependency update